cmake/OpenSimMacros.cmake

```diff
@@ -366,6 +366,8 @@ function(OpenSimAddTests)
             add_executable(${TEST_NAME} ${test_program}
                 ${OSIMADDTESTS_SOURCES})
             target_link_libraries(${TEST_NAME} ${OSIMADDTESTS_LINKLIBS})
+            target_include_directories(${TEST_NAME}
```

Comment:
Careful about transitively using some other dependency's dependency as a direct dependency in OpenSim. A special kind of hell can open up wherein the dependency gets updated and a bunch of entirely unrelated functionality breaks in downstream code.

Reply:
I'm not sure this is needed actually, if I include the `catch.hpp` using the direct path in testWrappingMatch.cpp. Removing it for now.